modules/instrument_builder/jsx/react.questions.js

```diff
@@ -0,0 +1,462 @@
+QuestionText = React.createClass({
```

Comment:
Can you add some documentation to make this easier to review? In particular, for each jsx file: 1. A comment at the top of the file describing what the components in the file are for and 2. A comment for each React component saying what that component is for, similarly to how we have that information in our PHP files?